sapai/data.py

```diff
@@ -8640,43 +8641,33 @@
             "baseHealth": 8,
             "packs": ["ExpansionPack1"],
             "level1Ability": {
-                "description": "Level-up: Gain +8/+8.",
-                "trigger": "LevelUp",
-                "triggeredBy": {"kind": "Self"},
+                "description": "Before Attack: Deal 6 damage to a random enemy once",
+                "trigger": "BeforeAttack",
+                "triggeredBy": {"kind": "Player"},
```

Comment:
Update `triggeredBy` to self for all levels of octopus. You can see the Boar for an example.